_posts/2023-06-23-run-and-debug-rust-verification-harnesses-natively-on-vscode.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title:  "Run and debug Rust verification harnesses natively on VS Code"
```

Comment:
Not sure about "natively", since Kani is being called through sub-processes. Do you have other ideas about the title? For example, ` The Kani VS Code Extension: Run and debug Rust verification harnesses from VS Code`

Reply:
I think `Introducing the Kani VSCode Extension` works as well.

Reply:
VS Code though. I do like the subtitle... it contains more information when sharing through social media etc